python/GafferSceneUITest/ParameterInspectorTest.py

```diff
@@ -291,7 +291,7 @@ def testSourceAndEdits( self ) :
 		self.__assertExpectedResult(
 			self.__inspect( s["independentLightTweak"]["out"], "/group/light", "intensity", s["editScope2"] ),
 			source = independentLightTweakPlug, sourceType = SourceType.Downstream,
-			editable = True, edit = lightEditScope2Edit, editWarning = "Parameter has edits downstream in independentLightTweak."
+			editable = True, edit = lightEditScope2Edit, editWarning = "Plug has edits downstream in independentLightTweak."
```

Comment:
The distinction between plugs/nodes and the data they produce is an important one, and I think is worth preserving here. Only the parameter (data in the scene flowing through the graph) can have downstream edits - the plug is just a plug and doesn't travel anywhere.

Perhaps we can solve this by adding this to the Inspector API, just above `name()`, and then using it when formatting the warning?

```
/// The type of property being inspected (for instance "attribute" or "parameter").
const std::string &type() const;
```